candi/bashible/detect_bundle.sh

```diff
@@ -32,12 +32,15 @@ if [ -e /etc/os-release ]; then
       >&2 echo "ERROR: Can't determine OS! No ID and VERSION_ID in /etc/os-release."
```

Comment:
Maybe we should also remove this? Is it possible that `/etc/os-release` doesn't exist?

Reply:
I don't know, but i think we can remove this if due to os-release inclusion won't success if file is absent.